README.md

```diff
@@ -14,10 +14,14 @@ CesiumJS is a JavaScript library for creating 3D globes and 2D maps in a web bro
 
 ### :rocket: Get Started
 
-Visit the [Downloads page](https://cesium.com/downloads/) or use the npm module:
+Visit the [Downloads page](https://cesium.com/downloads/) or install the [npm package](https://www.npmjs.com/package/cesium):
```

Comment:
I am a fan of the link to the cesium package on the npm registry, but I don't think we want the instructions for `yarn` in our README.md. We'd like to keep the instructions as simple as possible so the extra lines for yarn don't seem like they fit. 

The "```sh" is a nice addition though :+1:

Reply:
09b3826